ui/nuxeo-page-provider-display-behavior.js

```diff
@@ -49,7 +49,16 @@ export const PageProviderDisplayBehavior = [
 
       _computedEmptyLabel: {
         type: String,
-        computed: '_computeLabel(i18n, emptyLabel, filters, loading)',
+        computed: '_computeLabel(i18n, emptyLabel, filters, loading, size, _resultsCount)',
+      },
+
+      /**
+       * Total number of results in the page provider. Used to control the emptyLabel, because there might be a
+       * discrepancy between `size` and `resultsCount` right after the page provider finishes loading.
+       */
+      _resultsCount: {
+        type: Number,
+        value: -1,
```

Comment:
nit: we could consider making this property readonly, to prevent tampering from the outside

Reply:
This means that an empty table, which as the default size to 0, will always display "loading", because `_resultsCount` will always be -1 and this different than 0. I have triggered the preview so we can check [this case](https://preview-webui-fix-elements-1383-search-results-empty-label.webui.dev.nuxeo.com/?path=/story/ui-nuxeo-data-table--empty).

Reply:
🤔 You're right, this is a wrong behavior, as it should display `No results` instead. Thanks Gabe, I'll look into it.